contextproject-ms3/src/test/java/contextproject/models/Tracktest.java

```diff
@@ -0,0 +1,33 @@
+package contextproject.models;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Tracktest {
+
+  @Test
+  public void trackConstructorTest() {
+    URL resourceUrl = getClass().getResource("/beep.mp3");
+    Path resourcePath;
+    try {
+      resourcePath = Paths.get(resourceUrl.toURI());
+      Track tr = new Track(resourcePath.toString());
```

Comment:
In terms of naming things, I wouldn't go as far as shortening 'track' to 'tr'. Makes things less readable overall.